docs/hooks/hook_civicrm_alterEntityRefParams.md

```diff
@@ -0,0 +1,31 @@
+# hook_civicrm_alterEntityRefParams
+
+## Summary
+
+This hook is called when an `entityRef` field is rendered in a form, which allows you to modify the parameters used to fetch options for this kind of field.
+
+## Availability
+
+This hook is available in CiviCRM 4.7.27+ (maybe depends on the commit).
+
+## Definition
+
+     hook_civicrm_alterEntityRefParams(&$params, $formName)
+
+## Parameters
+
+- array `$params` - parameters of entityRef field
+- string `$formName` - form name
+
+## Example
+
+```php
+function myextension_civicrm_alterEntityRefParams(&$params) {
```

Comment:
@monishdeb i think this needs to have 

```php
, $formName
``` Also you need a $ in L25 for form name

Reply:
Oops... thanks @seamuslee001 for pointing that out